Mathlib/Topology/Instances/ENNReal.lean

```diff
@@ -1706,6 +1708,38 @@ lemma limsup_toReal_eq {ι : Type*} {F : Filter ι} [NeBot F] {b : ℝ≥0∞} (
   rw [key]
   rfl
 
+/-- If `xs : ι → ℝ≥0∞` is bounded, then we have `liminf (toNNReal ∘ xs) = toNNReal (liminf xs)`. -/
+lemma liminf_toNNReal_eq {b : ℝ≥0∞} (b_ne_top : b ≠ ∞)
```

Comment:
Isn't it enough to assume the weaker condition that `xs i` is eventually not equal to ` ∞`? If the liminf is infinity in this case, it means that the sequence is tending to infinity, so the liminf in NNReal is probably zero (what other choice of junk value could we have? I haven't checked).

Reply:
True. The existing lemma `Monotone.map_liminf_of_continuousAt` used here had suboptimal assumptions, so it looks like the right course is to generalize that first. I made PR #14591 for the generalization. With it, I will return and generalize this lemma (and `liminf_toReal_eq` above, which had a similar suboptimal assumption).

Reply:
I moved the generalization of this lemma and the ones below to #15115.